hare/builder.go

```diff
@@ -10,17 +10,19 @@ import (
 	"github.com/spacemeshos/go-spacemesh/signing"
 )
 
+//go:generate scalegen -types Message,Certificate,AggregatedMessages,InnerMessage
+
 // Message is the tuple of a message and its corresponding signature.
 type Message struct {
 	Sig      []byte
-	InnerMsg *innerMessage
+	InnerMsg *InnerMessage
```

Comment:
i meant that it should not be a pointer, so that it won't be encoded as an option

Reply:
I mean:
```
	if m.Message.InnerMsg == nil {
		logger.Error("onEarlyMessage called with nil message")
		return
	}

	if m.InnerMsg == nil {
		logger.Error("onEarlyMessage called with nil inner message")
		return
	}
```
what shall I replace it with if `InnerMsg` is not a pointer anymore?